epicfocus/README.md

```diff
@@ -0,0 +1,129 @@
+
+Many companies or individual product managers lack the means to prioritize and order their backlogs properly or have difficulties saying 'No' to their various stakeholders. As a result of these shortcomings, teams are tasked to work on everything at the same time. As time goes by, everything is in progress, and little is finished. Half done isn't done at all. The cost of such decisions is very high. Gerald Weinberg, in his book Quality Software Management: Systems Thinking, offers an illustration of the cost of context switching:
+
+![Context Switching](images/context-switching.png?raw=true "Waste percentage from switching context")
+
+Jeff Sutherland, in his book Scrum: The Art of Doing Twice the Work in Half the Time, speaks at length about damages of context switching. He calls waste a crime and encourages agile teams to do one thing at a time.
+
+Eureka! Every symptom we observed can be explained. It felt slow because of the lack of focus; everything was being worked on in parallel and took ages to complete. Teams didn't collaborate because there was no shared goal - the sprint backlog was a selection of unrelated stories and tasks. Nothing tangible could be shown during the review, as they had little progress in various directions. They were going everywhere and arriving nowhere.
+
+## Increase focus
+Acknowledgment of a problem is the first step towards its solution. There are several things you could do next. And at first, we aspired to give you some tips. But those seemed obvious and tricky at the same time. We wanted to tell you to focus, focus on the right stuff, but often it is easier said than done.
+
+We don't know your context and the reasons why this might be happening in your organization. Perhaps you're struggling to identify the right priority from many options, or maybe you have difficulties explaining to your stakeholders why you're putting their project on hold.
+
+So, instead of giving advice, we would like to ask you what you would do? How would you tackle this dysfunction? Would you address it at all?
+
+We strongly believe that by increasing focus and reducing context switching, you can expect a boost in productivity.
+
+## Dream of engagement
+Our story is complete. The symptoms we observed led us through different measurements to find a bottleneck that we can now solve. Our resources can be more productive. Numbers go up, stakeholders are happy.
+
+But are we happy? The resources we're dealing with they are human beings; even worse, they are knowledge workers. When we're dealing with people, we dream of increasing engagement. When trying to create more focus, we should start with a clear and compelling goal to give our teams a purpose.
+
+Create focus not to increase productivity alone, but to grow engagement as well. Now we need to look for a fourth metric to validate — the measure of engagement.
```

Comment:
Is this metric left for the next blog post? :)

Reply:
Exactly, this is the hook for the future :)

Reply:
Then I would give a slightly less subtle hint that a new blog post is coming, because I was kinda expecting to see this one continue to explain this new metric.

Reply:
Would it sound better if I change "Now" to "In the future"

Reply:
I think it's fine as it is. If I would change it my impulse would be to make it even more subtle, because I don't know when, or if, a follow-up blog is coming. For me, it's more a trigger to get the reader thinking about a metric like that.

A more subtle version could be:
`It would be great if we can come up with a fourth metric to validate — the measure of engagement.`